### PR TITLE
Added actor type: "Form authors supervisor"

### PR DESCRIPTION
**Pull request by aagz**
User from 'Responsible' field

### CODE REVIEW

File: inc/abstracttarget.class.php [hunk 1]
```diff
@@ -696,6 +696,17 @@ protected function prepareActors(PluginFormcreatorForm $form, PluginFormcreatorF
...
+
+            case PluginFormcreatorTarget_Actor::ACTOR_TYPE_AUTHORS_SUPERVISOR:
+               $requester_id = $formanswer->fields['requester_id'];
+
+               $user = new User;
+               $users = $user->find(['id' => $requester_id]);
+               if (!empty($users)) {
+                  $userIds = [$users[$requester_id]['users_id_supervisor']];
+               }
+               // $userIds = [$formanswer->fields['requester_id']];
```
Comment (btry) [inline inc/abstracttarget.class.php:708]: $user = User::getById($requester_id);
```suggestion
               $user = User::getById($requester_id);
               if ($user !== false) {
                  $userIds = [$user->fields['users_id_supervisor']];
               }
```

**Review by btry — COMMENTED** [review]
Looks good, but a part of code could be more readable. See my suggestion; I think it should work.

File: inc/abstracttarget.class.php (hunk 1 above)
Comment (aagz) [reply inc/abstracttarget.class.php:708]: Thanks, it works.

**Review by aagz — COMMENTED** [review]
Thank you.

**Comment by btry** [comment]
I rebase your contribution onto 2.12.0 because this is a small feature.

**Pull request merged**